.github/workflows/ccruntime_e2e.yaml

```diff
@@ -55,6 +55,13 @@ jobs:
         env:
           TARGET_BRANCH: ${{ inputs.target-branch }}
 
+      - name: Login to Confidential Containers ghcr.io
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
```

Comment:
@stevenhorsman I think that we need to create this secrets and have that info, is this something that you can help me to set up?

Reply:
I think the gitub.actor value is automatically generated based on the last person to modify the flow, rather than a secret